cns/service/main.go

```diff
@@ -667,6 +667,13 @@ func main() {
 		return
 	}
 
+	err = platform.SetMellanoxRegKeyPriorityVLANTag()
+	if err != nil {
+		logger.Errorf("Failed to set reg key for mellanox adapter: %v", err)
+	} else {
+		logger.Printf("Successfuly set reg key for existing Mellanox adapter.")
```

Comment:
actually, it will return the error in this case... 
`if adapatername == "" {
		return fmt.Errorf("no network adapter found with \"%s\" in description", mellanoxSearchString)
}`